src/applications/personalization/connected-accounts/components/NoConnectedApps.jsx

```diff
@@ -10,31 +10,45 @@ const recordAction = () => {
   });
 };
 
-export function NoConnectedApps({ errors, propertyName }) {
-  let title;
-  if (errors.length > 0) {
-    title = <h3>Sorry — we can’t seem to find any connected accounts.</h3>;
-  } else {
-    title = <h3>You do not have any connected accounts.</h3>;
+export class NoConnectedApps extends React.Component {
+  componentDidMount() {
+    this.header.focus();
   }
 
-  return (
-    <div className="row va-connected-acct-null">
-      <div className="usa-width-two-thirds medium-8 small-12 columns">
-        {title}
-        <div className="feature">
-          <h3>Have questions about signing in to {propertyName}?</h3>
-          <p>
-            Get answers to frequently asked questions about how to sign in,
-            common issues with verifying your identity, and your privacy and
-            security on {propertyName}.
-          </p>
+  noAccountsMessage() {
+    return this.props.errors.length > 0
+      ? 'Sorry — we can’t seem to find any connected accounts.'
+      : 'You do not have any connected accounts.';
+  }
+
+  render() {
+    return (
+      <div className="row va-connected-acct-null">
+        <div className="usa-width-two-thirds medium-8 small-12 columns">
+          <h3
```

Comment:
Does this `<h3>` get focused when the user has no connected accounts? I might pull code locally so I can test the handling in that situation.

Reply:
It does! This component is rendered instead of the `ConnectedApps` component when there are no connected accounts, so the h3 gets focused because of this code: https://github.com/department-of-veterans-affairs/vets-website/blob/c7bde59ef9a0fa81dd63b06caf7600fcbf3645de/src/applications/personalization/connected-accounts/components/NoConnectedApps.jsx#L14-L16